jenkins/performance-ci/performance-check.groovy

```diff
@@ -0,0 +1,140 @@
+/* groovylint-disable DuplicateStringLiteral, LineLength, NestedBlockDepth */
```

Comment:
should we name this file as `performance-ci.groovy` as it is ci for performance?

Reply:
The folder name is `performance-ci` hence used different names for files - https://github.com/shailesh-vaidya/cortx-re/tree/performance-ci-fix/jenkins/performance-ci
Also, this job is for a developer to validate performance on his custom codebase.

Reply:
okay